modules/tinymce/src/themes/silver/main/ts/ui/menus/menu/MenuUtils.ts

```diff
@@ -35,62 +50,56 @@ export const createHorizontalPartialMenuWithAlloyItems = (value: string, _hasIco
   };
 };
 
-// TODO: Potentially make this private again.
-export const createPartialMenuWithAlloyItems = (value: string, hasIcons: boolean, items: ItemTypes.ItemSpec[], columns: Toolbar.ColumnTypes, presets: Toolbar.PresetTypes): PartialMenuSpec => {
-  if (presets === 'color') {
+export const createPartialMenuWithAlloyItems = (value: string, hasIcons: boolean, items: ItemTypes.ItemSpec[], columns: Toolbar.ColumnTypes, menuLayout: MenuLayoutType): PartialMenuSpec => {
+  const getNormalStructure = (): StructureSpec => {
+    switch (menuLayout.menuType) {
```

Comment:
Any particular reason you use a switch here?

Reply:
Ah, now I see. It used to have more branches.

- [ ] change back to if.

Reply:
Updated: switched (haha) back to `if`.